### DISCUSSION TITLE
Welcome to esphome-junctek_kgf Discussions!

### DISCUSSION THREAD

**Discussion by tfyoung** [tfyoung/esphome-junctek_kgf#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jowser7** [comment]
Hi TF. Found your site, taking a look at the code. I have a Pipsolar build running on esphome and also a Junctek KG140f. So I was looking to keep pushing for more esphome sensors.  Are you sending your 4 values anywhere, or how are you using them once you parse and output them to serial?  
I am not much of a developer but can piece some things together. I like how Pipsolar built a webpage and MQTT support and am trying to figure out how to pull your readings into a page and topic like they have done.  https://github.com/syssi/esphome-pipsolar  What's your thoughts?   Cheers.

**Reply by tfyoung** [reply]
I mostly just get the voltage out to be honest. 

I haven't found the junctek battery percentage very accurate. I have my setup such that a small solar panel feeds the battery while the fridge drains it. If the voltage drops too low, I trigger a relay on my esp to turn the fridge over to mains power.  The cool thing is this whole automation is done with yaml.

I haven't found the need to add the other variables, though they are all parsed, so it's really just a matter of copying/pasting and typing. I'm happy to add a couple more if people have a need for them.

**Comment by AORDU** [comment]
Buenas tfyoung. Tengo un Junkect KG-100F y lo quiero comunicar a traves de un EspHome para disponer de la información en el HA, estoy domotizando mi autocaravana. Me podrías pasar un esquema, fotografías del conexionado desde el conector RJ10 (junteck) y pines de conexión del ESPHome. Te agradecería toda esta información.
Un Saludo. Antonio Orduña
